src/main/java/com/inorg/services/product/service/ProductServiceImpl.java

```diff
@@ -102,6 +101,12 @@ public List<Product> createProducts() {
                     List<ProductUpdateAction> updateActions = new ArrayList<>();
                     //TODO Create List of Update Actions
 
+                    ProductUpdateAction setDescription = ProductSetDescriptionActionBuilder.of()
```

Comment:
Add more Update Actions for updating:

Reply:
1. categories
2. name
3. variant price
4. variant image
5. variant attributes [size, color, brand]